section3/exercises/arrays.rb

```diff
@@ -77,11 +85,9 @@
 posts = ["image at beach", "holiday party", "adorable puppy", "video of cute baby"];
 
 # YOU DO: Think of a web application you commonly use. Where do you see LISTS utilized, where arrays
-# may be storing data? Come up with 3 examples - they could be from different web applications or 
+# may be storing data? Come up with 3 examples - they could be from different web applications or
 # all from the same one.
 
-# 1:
-# 2:
-# 3:
-
-
+# 1: Retail shopping websites.
+# 2: Would Spotify fulfill this? Has a list of artists, songs, albums, podcasts
+# 3: Character lists in a video game
```

Comment:
✅

Reply:
Spotify (and other example), probably use a large combination of data types, but I could see a list of artists being an array!